docs/how_to_create_a_plugin.md

```diff
@@ -1,11 +1,12 @@
 # How to create a Decidim engine
 
 1. Run the following command:
+
 ```bash
 - rails plugin new decidim-<engine_name>
 ```
 
-2. Change the `decidim-<engine_name>.gemspec` file:
+1. Change the `decidim-<engine_name>.gemspec` file:
```

Comment:
Thanks! Should be fixed now, I forgot to fix it because I wanted to get feedback on the change first since I'm not sure I even like it myself... :)